server/matchmaker/search.py

```diff
@@ -247,13 +247,9 @@ def _players_repr(self) -> str:
 
     def __repr__(self) -> str:
         """For debugging"""
-        string = f"Search({[p.login for p in self.players]}, {self.average_rating}"
-        if self.failed_matching_attempts > 0:
-            string += f", FMA: {self.failed_matching_attempts}"
-        if self.has_newbie():
-            string += ", has_newbie"
-        string += ")"
-        return string
+        return f"Search({[p.login for p in self.players]}, {self.average_rating}" \
```

Comment:
Don't need the backslash at the end of the line

Reply:
Without the backslash the following code becomes unreachable because of the return statement.

Reply:
Oh, then put it in parenthesis. Like this:

```python
return (
    f"some string {stuff}. "
    r"This part is a raw string, "
    "and this part is a normal string"
)
```
That would become
```python
"some string value_of_stuff. This part is a raw string, and this part is a normal string"
```